src/styles/globals.css

```
@@ -518,6 +518,40 @@ input::placeholder {
     animation: slideInRight 0.3s ease-out forwards;
 }
 
+/* Cycling loading word + dots (used by CyclingLoading) */
+@keyframes cyclingFadeIn {
```

Stylelint (17.11.0)

[error] Expected keyframe name "cyclingFadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)